src/main/java/model/order/AdditionalInfo.java

```diff
@@ -0,0 +1,10 @@
+package model.order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AdditionalInfo {
```

Comment:
What is the use of this class ?

Reply:
it will print customer id

Reply:
@nehagandhi24 Remove this class. We are not going to it in Synkron